packages/api-openfin/notification.html

```diff
@@ -0,0 +1,11 @@
+<h5 id="message">Message</h5>
```

Comment:
AFAIK that's just the way Chrome handles minimal HTML5 documents. It adds in the `html`, `head`, `body` elements in the correct locations.